www/source/demo/windows/steps/9.html.slim

```diff
@@ -0,0 +1,138 @@
+section
+ h1 Connect to a Database
+
+ p This topic covers the following areas:
+
+ ul
+  li #{link_to 'Setting service bindings','#setting-bindings'} 
+  li #{link_to 'Defining database connection logic','#connection-logic'}
+  li #{link_to 'Connecting to a database running inside the Habitat Studio','#connect-studio'}
+  li #{link_to 'Connecting to a database running inside a Docker container','#connect-docker'}
+
+ h2 <a name="setting-bindings">Setting service bindings</a>
+
+ p Service bindings allows runtime configuration values for one service group to be shared with another service group, forming a producer/consumer relationship. When you want to share out configuration data with another service group, you must add the <code>pkg_exports</code> setting to your plan.
+ 
+ p The following shows how a <code>plan.ps1</code> exports a port key for consuming services, such as a reverse proxy or load balancer.
+  
+ = code(:bash) do
+  |
+   $pkg_exports=@{"port"="port"}
+
+ p This will expose the following from the default.toml:
+
+ = code(:toml) do
+   |
+    port = 8099
+
+
+ p Conversely, if your web application wants to bind to a key, such as a port from a database, you need to either use the <code>pkg_binds</code> or <code>pkg_binds_optional</code> settings in your plan. This requires that you know what keys are exported by the producer service group.
+
+ p You can find exported keys by searching for the package at #{link_to 'https://bldr.habitat.sh/#/pkgs/core', 'https://bldr.habitat.sh/#/pkgs/core'} and viewing the manifest for a given release of a package.
+
+ .screenshot
+    img src="/images/screenshots/exports_callout.png"
```

Comment:
As I read through this, it was unclear at first that this was an image, and then once i realized it was, I had to think about what it was referring to. It might help to link to an example package and then indicate that the exported config can be found under **Plan Source**, or use a screenshot that calls out the important stuff in context, like maybe:

<img width="1200" alt="image" src="https://user-images.githubusercontent.com/274700/42003102-9fa08e54-7a1e-11e8-8937-5ec37696b19e.png">